app/models/user.rb

```diff
@@ -0,0 +1,35 @@
+require 'bcrypt'
+require 'dm-validations'
+
+
+class User
+  include DataMapper::Resource
+
+  attr_reader :password
+  attr_accessor :password_confirmation
+
+  property :id, Serial
+  property :first_name, String
+  property :last_name, String
+  property :email, String
+  property :team, String
+  property :password_digest, Text
+  validates_uniqueness_of :email
+
+  validates_confirmation_of :password, message: 'Sorry, your passwords do not match'
```

Comment:
Line is too long. [84/80]<br>Prefer the new style validations `validates :column, confirmation: value` over `validates_confirmation_of`.